tests/conftest.py

```
@@ -18,446 +19,463 @@
+        mock_instance.decrypt_for_user = AsyncMock(return_value="ghp_1234567890abcdef")
+        mock_instance.encrypt = MagicMock(return_value=TOKEN_ENCRYPTED_1)
+        mock_instance.decrypt = MagicMock(return_value="ghp_1234567890abcdef")
+
+        # Make the class mock return the instance when called
+        mock_helper_class.return_value = mock_instance
+
+        yield mock_helper_class
 
 
 @pytest.fixture
 def sample_token_id():
```

Ruff (0.11.9)

Redefinition of unused `sample_token_id` from line 333

(F811)